utils/release/git-publish-all.mjs

```diff
@@ -31,9 +35,38 @@ const getCliChangelog = () => {
 
 // Commit, tag and push
 (async () => {
-  const REPO_OWNER = 'coveo';
-  const REPO_NAME = 'cli';
   const PATH = '.';
+  const GIT_USERNAME = 'developer-experience-bot[bot]';
+  const GIT_EMAIL =
+    '91079284+developer-experience-bot[bot]@users.noreply.github.com';
+  const GIT_SSH_REMOTE = 'deploy';
+  //#endregion
+
+  // #region Setup Git
+  await gitSetupSshRemote(
+    REPO_OWNER,
+    REPO_NAME,
+    process.env.DEPLOY_KEY,
+    GIT_SSH_REMOTE
+  );
+  await gitSetupUser(GIT_USERNAME, GIT_EMAIL);
+  // #endregion
+
+  //#region GitHub authentication
+  const authSecrets = {
+    appId: process.env.RELEASER_APP_ID,
+    privateKey: process.env.RELEASER_PRIVATE_KEY,
+    clientId: process.env.RELEASER_CLIENT_ID,
+    clientSecret: process.env.RELEASER_CLIENT_SECRET,
+    installationId: process.env.RELEASER_INSTALLATION_ID,
+  };
+
+  const octokit = new Octokit({
+    authStrategy: createAppAuth,
+    auth: authSecrets,
+  });
```

Comment:
I think it's important to keep both very separated because its two very different authentication type, two different permissions set.
The distinction is important to keep.

However, we do not use the SSH here, so I removed it